util/topgen/merge.py

```diff
@@ -3,13 +3,52 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import logging as log
+import random
 from copy import deepcopy
 from functools import partial
 from collections import OrderedDict
+from math import ceil, log2
 
 from topgen import lib
 
 
+def modname_to_pascal_case(name):
```

Comment:
just curious, do you think there's any way to reuse the code [here?](https://github.com/lowRISC/opentitan/blob/master/util/topgen/c.py#L14)  I've been thinking for awhile it would be kind of nice to centralize all the naming stuff.. (probably a different PR though)

Reply:
oh good idea, I haven't seen that. will move to that one.

Reply:
There's another code out there converting snake_case to PascalCase don't remeber where though.... :(

Reply:
yeah I guess we have that in several places by now ;)... I reused `c.py` for now.